modules/nw-dns-forward.adoc

```diff
@@ -16,7 +25,7 @@ You can use DNS forwarding to override the forwarding configuration identified i
 $ oc edit dns.operator/default
 ----
 +
-This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.
+This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the upstream DNS servers.
```

Comment:
```suggestion
This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the upstream DNS servers under `UpstreamResolvers`.
```

Reply:
Looks like this suggested change didn't make it in.

Reply:
Since `UpstreamResolvers` comes later in the procedure, it may be out of place. Just "upstream DNS servers" conveys the same.

Reply:
Did you confirm this with your SME?